src/main/fc/rc_adjustments.h

```diff
@@ -75,8 +75,10 @@ typedef enum {
     ADJUSTMENT_VEL_Z_D                  = 47,
     ADJUSTMENT_FW_MIN_THROTTLE_DOWN_PITCH_ANGLE = 48,
     ADJUSTMENT_VTX_POWER_LEVEL = 49,
+    ADJUSTMENT_TPA                      = 50,
+    ADJUSTMENT_TPA_BREAKPOINT           = 51,
```

Comment:
Can we add it after "ADJUSTMENT_PROFILE " so that existing entries do not change id?

Reply:
Unfortunately no, the configurator generates the adjustment on the fly, so the ids have to be without a gap, and the inactive element ADJUSTMENT_PROFILE have to be the last.

Reply:
What is the way forward for this, as I am looking to add an adjustment also? It seems that if PROFILE_ADJUSTMENT can be inactive, there is no way it can maintain an ID, unless configurator treats adjustments like modes, and receives the IDs too.

Reply:
@DzikuVx @Scavanger I don't see `ADJUSTMENT_PROFILE` being handled anywhere in the code, so to me it looks like this was added to the list of the available adjustments in 0ca01635a6610f791d98c22d4cf5bedb49dda3f1 and never actually implemented. I think it's safe to remove `ADJUSTMENT_PROFILE` altogether from both firmware and Configurator

Reply:
FYI, I did not see it in Configurator at all.

Reply:
I'm going to merge this and clean up the `ADJUSTMENT_PROFILE` in a subsequent PRs to firmware and Configurator.